stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -1,57 +1,81 @@
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: alnwf !< mean nir albedo with weak cosz dependency
   !emissivity
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: semis !< ???
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: semis !< surface lw emissivity in fraction
   !roughness length for land
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: zorll !< ???
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: zorll !< land surface roughness in cm
 
   !real(kind=kind_phys), dimension(:,:),   allocatable, save :: stype
-  integer, dimension(:,:),   allocatable, save :: stype !< ???
+  integer, dimension(:,:),   allocatable, save :: stype !< soil type
 
   ! For cellular automata
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: sst !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: lmsk !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: lake !< ???
-  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: uwind !< ???
-  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: vwind !< ???
-  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: height !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: dx !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: condition !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca_deep_cpl !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca_turb_cpl !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca_shal_cpl !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca1_cpl !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca2_cpl !< ???
-  real(kind=kind_phys), dimension(:,:),   allocatable, save :: ca3_cpl !< ???
+  !> Sea surface temperature (SST) array, allocated dynamically.
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: sst 
+  !> Logical mask array for stochastic physics processes
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: lmsk 
+  !> A 2D real array to store lake data, with kind specified by kind_phys.
+  !> The array is allocatable and saved across subroutine calls.
+  real(kind=kind_phys), dimension(:,:),   allocatable, save :: lake 
+  !> Allocatable array to store the u-component of wind (zonal wind) 
+  !> with dimensions (latitude, longitude, vertical levels).
+  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: uwind 
+  !> Allocatable array to store the v-component of wind (velocity in the y-direction)
+  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: vwind 
+  !> Array to store the height values with physical kind precision.
+  real(kind=kind_phys), dimension(:,:,:), allocatable, save :: height 
+  !> 2D real array to store the stochastic perturbations, allocated dynamically and saved across subroutine calls
```

Comment:
Should double check this, is dx not the 2-dim grid spacing variable (how large is each grid cell)?